src/arrayMethodSort.js

```diff
@@ -5,7 +5,11 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    if (typeof compareFunction === 'function') {
+      return Array.prototype.sort.call(this, compareFunction);
```

Comment:
Instead of returning the native Array.prototype.sort, implement a custom sorting method.

Reply:
A helper function to compare two elements should be created and used